Assets/Scripts/PowerUps/PowerUpBase.cs

```diff
@@ -29,7 +29,8 @@ public void Interact(ICanInteract interactor)
 
     private void PlayPickUpSound()
```

Comment:
This function feels like it is not correctly named

Reply:
Yea gonna name it to something else naoo